app/uk.gov.hmrc.helptosaveproxy/config/AppConfig.scala

```diff
@@ -40,4 +40,7 @@ object AppConfig extends ServicesConfig {
 
   val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}/nsi-services/account"
 
+  def dwpUrl(nino: String, transactionId: UUID): String =
+    s"${baseUrl("dwp")}/v1/isEligible/$nino?systemId=607&thresholdAmount=650&transactionId=$transactionId"
```

Comment:
the threshold amount and the system ID should come from config as well

Reply:
done, thank you